build_runner/README.md

```diff
@@ -25,6 +25,10 @@ Dart code, outside of tools like `pub`. Unlike `pub serve/build`, files are
 always generated directly on disk, and rebuilds are _incremental_ - inspired by
 tools such as [Bazel][].
 
+> **NOTE**: Are you are user of this package? You may be interested in
+> simplified user-facing documentation, such as our
+> [getting started guide](docs/getting_started.md).
```

Comment:
I don't think this link will work on the package site – yet?

Reply:
+1